<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="scenery1"
           tests="2"
           failures="1"
           errors="0"
           skipped="0"
           time="210.477">
  <properties />
            <testcase name="test现金流量表" time="12.584" classname="com.dzf.test.testscenery.TestScenery1">
                <failure type="com.dzf.test.util.MyException"
                  message="【打印面板-打印按钮】 点击失败！"
              >
<![CDATA[com.dzf.test.util.MyException: 【打印面板-打印按钮】 点击失败！
  com.dzf.test.model.Handler.click(Handler.java:191)
  com.dzf.test.page.accounting.财务账表.现金流量表Page.print(现金流量表Page.java:53)
  com.dzf.test.testscenery.TestScenery1.test现金流量表(TestScenery1.java:438)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:647)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:824)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1132)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:778)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1222)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1147)
  org.testng.TestNG.runSuites(TestNG.java:1072)
  org.testng.TestNG.run(TestNG.java:1044)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: org.openqa.selenium.TimeoutException: Timed out after 5 seconds waiting for element to be clickable: [[FirefoxDriver: firefox on WINDOWS (da29989a-a8a9-4d95-bf9a-bdebbe34bcbf)] -> xpath: .//a[@onclick='doPrint()']]
Build info: version: '2.41.0', revision: '3192d8a6c4449dc285928ba024779344f5423c58', time: '2014-03-27 11:29:39'
System info: host: 'dwk-pc', ip: '192.168.56.1', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_77'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=19.0, platform=WINDOWS, browserConnectionEnabled=true, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: da29989a-a8a9-4d95-bf9a-bdebbe34bcbf
  org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
  org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:261)
  com.dzf.test.model.Handler.click(Handler.java:180)
  com.dzf.test.page.accounting.财务账表.现金流量表Page.print(现金流量表Page.java:53)
  com.dzf.test.testscenery.TestScenery1.test现金流量表(TestScenery1.java:438)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:647)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:824)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1132)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:778)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1222)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1147)
  org.testng.TestNG.runSuites(TestNG.java:1072)
  org.testng.TestNG.run(TestNG.java:1044)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="test期初试算平衡" time="197.893" classname="com.dzf.test.testscenery.TestScenery1" />
      
</testsuite>
